docs/getting-started/automatic-import.asciidoc

```diff
@@ -65,6 +79,7 @@ image::images/auto-import-success-message.png[The automatic import success messa
 +
 14. Click **Add to an agent** to deploy your new integration and start collecting data, or click **View integration** to view detailed information about your new integration. 
 
-NOTE: Once you've added an integration, you can't edit any details other than the ingest pipeline, which you can edit by going to **Stack Management → Ingest Pipelines**. 
+
+TIP: Once you've added an integration, you can edit the ingest pipeline by going to **Project Settings → Stack Management → Ingest Pipelines**.
```

Comment:
I think this can function as an optional step or as a tip. If you keep it as a tip, you might want to combine it with the one that's directly following it, so users don't have to read two separate tips.